### DISCUSSION TITLE
Slideshow from JupyterLab notebooks

### DISCUSSION THREAD

**Discussion by fjhickernell** [jupyterlab/jupyterlab#14942] [Q&A]
As a beginner to Jupyter and JupyterLab, I am curious about how to create a slideshow from my notebook.  In the past I have used LaTeX-Beamer for presentations, but a future presentation will contain a lot of Python code that I want to execute, so I am exploring options with JupyterLab.  RISE seems not yet to work for JupyterLab.  What suggestions do you have?  Thanks.

**Comment by krassowski** [comment]
Did you try https://github.com/jupyterlab-contrib/rise?

**Reply by fjhickernell** [reply]
Yes, but it does not work as expected.  I do not see the tabs on the cells that allow me to mark them as slides, sub-slides, etc.  The code in the slides is not executable.  Any suggestions?

**Reply by fjhickernell** [reply]
Now I see the tabs under Common Tools on the right, but the code is still not executable when it is changed in presentation mode.  Help please.
